src/main/java/com/fasterxml/jackson/databind/util/EnumResolver.java

```diff
@@ -126,7 +126,27 @@ public static EnumResolver constructUsingToString(DeserializationConfig config,
                 config.isEnabled(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS));
     }
 
-    /**
+
+   public static EnumResolver constructUsingIndex(DeserializationConfig config, Class<Enum<?>> enumCls) {
```

Comment:
indent looks wrong at least on github

Reply:
user seems to be using 3 space indents instead of 4 space indents that are used in existing code

Reply:
i don't have to visit my optometrist then :sweat_smile:

Reply:
😂 My bad for the visual trickery

Reply:
@pjfanning @yawkat
Just to make sure, is it 4 space indents? I think I references the wrong classes for indents